WordPress/src/main/java/org/wordpress/android/ui/prefs/MyProfileFragment.java

```diff
@@ -0,0 +1,168 @@
+package org.wordpress.android.ui.prefs;
+
+import android.app.Fragment;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import org.wordpress.android.R;
+import org.wordpress.android.models.Account;
+import org.wordpress.android.models.AccountHelper;
+import org.wordpress.android.models.AccountModel;
+import org.wordpress.android.util.NetworkUtils;
+import org.wordpress.android.util.StringUtils;
+import org.wordpress.android.widgets.WPTextView;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import de.greenrobot.event.EventBus;
+
+public class MyProfileFragment extends Fragment {
+    private final String DIALOG_TAG = "DIALOG";
+
+    private WPTextView mFirstName;
+    private WPTextView mLastName;
+    private WPTextView mDisplayName;
+    private WPTextView mAboutMe;
+
+    public static MyProfileFragment newInstance() {
+        return new MyProfileFragment();
+    }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+
+        refreshDetails();
+        if (NetworkUtils.isNetworkAvailable(getActivity())) {
+            AccountHelper.getDefaultAccount().fetchAccountSettings();
+        }
+    }
+
+    @Override
+    public void onStart() {
+        super.onStart();
+
```

Comment:
Nitpick, let's remove the empty line or add one in `onStop()` for symmetry. Thanks!